src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
will be set to 0 by default

Reply:
done